### ISSUE TITLE
minecraft 1.12.2 / overviewer  0.12.204 not loading textures

### ISSUE THREAD

**Issue by wait-io** [overviewer/Minecraft-Overviewer#1418]
Hi,

I've been running overviewer for over a year now, most updates went smoothly but now I'm lost.

Running  overviewer 0.12.204 (22b9ab3) and minecraft 1.12.2 vanilla on MineOS (debian).
Updated to 1.12 and 1.12.1 later on. Map generation just stopped working. No matter what I do, it won't find the .jar file. It's in the same directory structure as before (updates are done with scripts), nothing changed and I can't get it to work.
Even used "texturepath" in the map.config I'm using.

error is
`overviewer.py:481  12010 2017-09-26 22:03:31 INFO     Generating textures...                                                                                            overviewer.py:628  12010 2017-09-26 22:03:33 ERROR    Could not find the textures while searching for 'assets/minecraft/textures/blocks/grass_top.png'. Try specifying the 'texturepath' option in your config file.                                                                                                                            Set it to the path to a Minecraft Resource pack.
Alternately, install the Minecraft client (which includes textures)
Also see <http://docs.overviewer.org/en/latest/running/#installing-the-textures>
(Remember, this version of Overviewer requires a 1.12-compatible resource pack)
(Also note that I won't automatically use snapshots; you'll have to use the texturepath option to use a snapshot jar)`

paths and files
`mc2 ~/.minecraft/versions # tree
.
├── 1.12
│   └── 1.12.jar
├── 1.12.1
│   └── 1.12.1.jar
└── 1.12.2
    └── 1.12.2.jar

3 directories, 3 files`

I hope I just overlooked something stupid. 

Thanks,
Daniel

**Comment by CounterPillow** [comment]
Is that the full content of the versions directory? We've had an issue before where the April Fools 2.0 update was causing issues

**Comment by wait-io** [comment]
Yes. I cleaned up after the 1.12 release because of the same issue. I tried leaving only a single version in there as well as all three 1.12* releases. No change.

**Comment by CounterPillow** [comment]
Can you `sha1sum 1.12.2.jar` for me so that I can confirm that your file isn't corrupt in some way?

**Comment by triplealpha** [comment]
FYI - I resolved this issue by specifying a texturepath to the 1.10.2 jar file.  ( I'm using overviewer-0.12.137, and the error I got mentioned that version requires 1.10 compatible texture pack. )

Suggestion for OP, specify a texturepath directly to the 1.12 jar? (not 1.12.1 or 1.12.2).

**Comment by CounterPillow** [comment]
@triplealpha that's a completely different issue (namely your overviewer being out of date)

**Comment by bryanloeper** [comment]
I ran into the same issue just now - `0.12.137` is the latest release available on the [downloads page](https://overviewer.org/downloads) for Windows. Looks like Fedora is out of date too.

**Comment by triplealpha** [comment]
> that's a completely different issue (namely your overviewer being out of date)

True, I was posting mostly because google is going to lead tons of people with similar but not identical issues to this issue (as it did for me), and my own issue/solution would help them as opposed to them opening a new issue.

I guess for Devs this isn't an "open forum", but rather a literal "fix this specific bug" workflow :)
